test/get/path/shared_dependency_test.dart

```diff
@@ -73,38 +71,4 @@ main() {
     await d.appPackagesFile(
         {"foo": "../foo", "bar": "../bar", "shared": "../shared"}).validate();
   });
-
```

Comment:
What happened to these tests?

Reply:
I explained in [the commit message](https://github.com/dart-lang/pub/pull/1842/commits/38f376e885ac88a6288662cafce63f2030acc6de).